examples/logging/foldersink.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: logging.gcp.upbound.io/v1beta1
+kind: FolderSink
+metadata:
+  annotations:
+    meta.upbound.io/example-id: logging/v1beta1/foldersink
+    upjet.upbound.io/manual-intervention: "FolderSink requires a folder id"
+  labels:
+    testing.upbound.io/example-name: my-sink
+  name: my-sink
+spec:
+  forProvider:
+    description: some explanation on what this is
+    destination: storage.googleapis.com/${google_storage_bucket.log-bucket.name}
+    filter: resource.type = gce_instance AND severity >= WARNING
+    folderSelector:
+      matchLabels:
+        testing.upbound.io/example-name: my-folder
+
+---
+apiVersion: cloudplatform.gcp.upbound.io/v1beta1
+kind: Folder
+metadata:
+  annotations:
+    meta.upbound.io/example-id: logging/v1beta1/foldersink
+    upjet.upbound.io/manual-intervention: "Requires organization level IAM permissions"
+  labels:
+    testing.upbound.io/example-name: my-folder
+  name: my-folder
+spec:
+  forProvider:
+    displayName: My folder
+    parentSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+
+---
+apiVersion: storage.gcp.upbound.io/v1beta1
+kind: Bucket
+metadata:
+  annotations:
+    meta.upbound.io/example-id: logging/v1beta1/foldersink
```

Comment:
```suggestion
    meta.upbound.io/example-id: logging/v1beta1/foldersink
    upjet.upbound.io/manual-intervention: "This resource is the dependency of FolderSink.logging. This resource is skipping because parent resource was skipped."
```